src/main/kotlin/org/flockhive/db/repos/UserRepo.kt

```diff
@@ -0,0 +1,62 @@
+package org.flockhive.db.repos
+
+import org.flockhive.db.models.User
+import org.flockhive.db.tables.users
+import org.flockhive.db.utility.Hasher
+import org.ktorm.database.Database
+import org.ktorm.dsl.eq
+import org.ktorm.entity.add
+import org.ktorm.entity.count
+import org.ktorm.entity.find
+import org.ktorm.entity.update
+
+class UserRepo(private val db: Database): Repo(db) {
+    fun find(id : Int): User? = db
+        .users.find { it.id eq id }
```

Comment:
```suggestion
        .users
        .find { it.id eq id }
```

It's not a one liner, because it's split on `db` but it also doesn't split all the way to make for easy read.